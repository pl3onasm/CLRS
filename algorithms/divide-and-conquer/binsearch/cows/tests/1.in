12
3 1 4 15 8 6 65 10 35 50 24 41 60 30 52 72 20 68 16 88 