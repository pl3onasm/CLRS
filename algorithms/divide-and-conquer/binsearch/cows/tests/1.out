Minimum distance: 5
Stall positions: 1 6 15 20 30 35 41 50 60 65 72 88 
